app/src/main/java/com/github/se/project/ui/profile/CreateUserProfile.kt

```diff
@@ -113,7 +143,11 @@ fun CreateProfileScreen(
 
               OutlinedTextField(
                   value = lastName,
-                  onValueChange = { lastName = it },
+                  onValueChange = {
+                    if (it.length <= context.resources.getInteger(R.integer.user_last_name)) {
+                      lastName = it
```

Comment:
What happens if someone tries to go over the cap? Does the typing just not change anything?

Reply:
Yes exactly! I thought it was the easier thing to do, send a warning would maybe be too much warning and a bit confusing.